jenkins/ci.suse.de/cloud-mkphyscloud-qa-scenario-1b.yaml

```diff
@@ -12,11 +12,13 @@
          - shared device for DB and rabbit
          - swift
          - 2 kvm and 2 xen compute nodes
+         - Neutron: linuxbridge or OVS
 
       It will wipe all machines in the selected cloud!
 
     wrappers:
-      - mkphyscloud-qa-common-wrappers
+      - build-name:
+          name: '#${BUILD_NUMBER} ${ENV,var="cloudsource"} - qa${ENV,var="hw_number"} ${ENV,var="networkingplugin"}:${ENV,var="networkingmode"}'
```

Comment:
This line seems to me a good candidate for a new wrapper in `macros.yaml `, I noticed it's being used for other scenarios as well.